### PR TITLE
Feature/issue 355 rebuild projects

### LINKED ISSUE #355: Aggiornare implementazione command rebuild_projects
## Dettagli

Il comando `rebuild_projects` permette di reindicizzare i documenti in base al publisher o al documento. Nella versione 3.7 + il task su cui fa affidamento `rebuild_projects` non esiste più e deve quindi essere reimplementato sulla base della codebase attuale.

`readthedocs.core.management.commands.update_repos` / `readthedocs.core.management.commands.update_versions` sono dei buoni punti di partenza, ma è comunque necessario integrare la gestione dei publisher / project

### PR DESCRIPTION
**Pull request by Amanit**
rebuild-projects management command fixed

Fix #355

### CODE REVIEW

File: readthedocs/projects/tasks.py [hunk 1]
```diff
@@ -967,6 +968,26 @@ def setup_python_environment(self):
         self.python_env.install_core_requirements()
         self.python_env.install_requirements()
 
+        # old version
```
Comment (yakky) [inline readthedocs/projects/tasks.py]: I think we should remove this

File: readthedocs/projects/tasks.py [hunk 2]
```diff
@@ -275,6 +275,7 @@ def run(self, version_pk):  # pylint: disable=arguments-differ
 # application itself (RTD code didn't failed). These exception are logged as
 # ``INFO`` and they are not sent to Sentry.
 @app.task(
+    queue='docs',
```
Comment (yakky) [inline readthedocs/projects/tasks.py]: the `docs` queue must be applied in our version of the command

File: readthedocs/docsitalia/management/commands/rebuild_projects.py [hunk 3]
```diff
@@ -58,21 +57,26 @@ def handle(self, *args, **options):
...
         for version in versions:
-            task = UpdateDocsTask()
+            task = update_docs_task
             build = Build.objects.create(
                 project=version.project,
                 version=version,
                 type='html',
                 state='triggered',
             )
+            # corrected version
```
Comment (yakky) [inline readthedocs/docsitalia/management/commands/rebuild_projects.py]: I think we can remove this command

File: readthedocs/core/management/commands/update_repos.py [hunk 4]
```diff
@@ -81,8 +74,6 @@ def handle(self, *args, **options):
                     log.info('Updating all internal versions for %s', slug)
                     for version in Version.internal.filter(
                             project__slug=slug,
-                            active=True,
-                            uploaded=False,
```
Comment (yakky) [inline readthedocs/core/management/commands/update_repos.py]: could you restore these two lines? I removed them by mistake

**Review by yakky — CHANGES_REQUESTED** [review]

**Review by yakky — APPROVED** [review]

**Review by pauloxnet — APPROVED** [review]

**Pull request merged**